tests/managed-service/test_post_installation_state.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources import pod
+from ocs_ci.framework.testlib import (
+    managed_service_required,
+    ManageTest,
+    tier1,
+)
+from ocs_ci.ocs.exceptions import CommandFailed
+
+log = logging.getLogger(__name__)
+
+
+class TestPostInstallationState(ManageTest):
+    """
+    Post-installation tests for ROSA and OSD clusters
+    """
+
+    @tier1
+    @pytest.mark.polarion_id("OCS-2694")
+    @managed_service_required
+    def test_deployer_logs_not_empty(self):
+        """
+        Test that the logs of manager container of ocs-osd-controller-manager pod are not empty
+        """
+        deployer_pod = pod.get_pods_having_label(
+            constants.MANAGED_CONTROLLER_LABEL, constants.OPENSHIFT_STORAGE_NAMESPACE
+        )[0]
+        deployer_logs = pod.get_pod_logs(
+            pod_name=deployer_pod["metadata"]["name"], container="manager"
+        )
+        log_lines = deployer_logs.split("\n")
+        for line in log_lines:
+            if "ERR" in line:
+                log.info(f"{line}")
+        log.info(f"Deployer log has {len(log_lines)} lines.")
+        assert len(log_lines) > 100
+
+    @tier1
+    @pytest.mark.polarion_id("OCS-2695")
+    @managed_service_required
+    def test_connection_time_out(self):
+        """
+        Test that connection from mon pod to external domain is blocked and gets timeout
+        """
+        mon_pod = pod.get_mon_pods()[0]
+        try:
+            mon_pod.exec_cmd_on_pod("curl google.com")
+            assert False, "Curl command should have failed due to timeout"
+        except CommandFailed as cmdfailed:
+            assert "Connection timed out" in str(cmdfailed)
```

Comment:
It would be more readable if this is expressed via [`pytest.raises()`](https://docs.pytest.org/en/6.2.x/reference.html?highlight=raises#pytest-raises)

Reply:
Thank you, fixed.